lib/pymedphys/_experimental/streamlit/utilities/icom.py

```diff
@@ -266,12 +266,8 @@ def _determine_width_length_centre(mlc, jaw):
     width = pd.Series(np.sum(mean_mlc, axis=1), name="width")
     length = pd.Series(jaw[:, 0] + jaw[:, 1], name="length")
 
-    centre_x = pd.Series(
-        (-mean_mlc[:, 0] + mean_mlc[:, 1]) / 2, name="experimental_icom_centre_x"
-    )
-    centre_y = pd.Series(
-        (-jaw[:, 0] + jaw[:, 1]) / 2, name="experimental_icom_centre_y"
-    )
+    centre_x = pd.Series((mean_mlc[:, 0] - mean_mlc[:, 1]) / 2, name="centre_x")
```

Comment:
deliberate negative of previous commit?

Reply:
Yup, previously I had just added in the result with a sort of "note to self experimental flag, I haven't tested if the coords are right here". Once I took some data, I had the x and y sign the wrong way round.